security/medusa/l2/acctype_ipc_msgrcv.c

```diff
@@ -8,29 +18,30 @@
 
 struct ipc_msgrcv_access {
 	MEDUSA_ACCESS_HEADER;
-	long m_type;	/* message type;  see 'struct msg_msg' in include/linux/msg.h */
+	long m_type;	/* message type; see 'struct msg_msg' in include/linux/msg.h */
 	size_t m_ts;	/* msg text size; see 'struct msg_msg' in include/linux/msg.h */
-	/* TODO char m_text[???]; send also message text? */
-	pid_t target; /* TODO: namespaces implementation??? */
+	/* char m_text[???]; message text is not sent, for now */
```

Comment:
For now, there is no reason send to authorization server content of the message.
IPC - inter process communication - provides following mechanisms: 1) semaphores, 2) shared memory segments, and 3)  message queues.
In the case of 3) we send to authorization server only control information (target of the message, message size etc.), not the message itself.
Why we don't send message itself? A message has not constant size. In our model (Medusa communication protocol) there is (for now) no possibility send an kobject with variable size. But, one day... TODO is for this day ;)

Reply:
I appreciate that you thought ahead about the problem. However, if it is something that is important, we will know that it has to be done, there is no way we will overlook it. If we forget, it means that it would be something obsolete anyway. I believe here applies YAGNI principle (https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it). 

`` But, one day... TODO is for this day ;) ``
That day may not come for countless reasons - the kernel may change, the Medusa infrastructure may change, the purpose of the project may change or we simply decide that it is not needed anymore etc.